ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/mac_address_table.schema.yml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  mac_address_table:
+    documentation_options:
+      filename: Fabric Variables
+      table: MAC Address Table Settings
+    type: dict
+    description: |
+      MAC address-table aging time
+      Use to change the EOS default of 300.
+    keys:
+      aging_time:
```

Comment:
Should we had min and max?

from vEOS:
```
Leaf1(config)#mac address-table aging-time ?
  <0-0>         Enter 0 to disable aging
  <10-1000000>  Aging time in seconds
```

Reply:
addressed in [76813b1](https://github.com/aristanetworks/ansible-avd/pull/2510/commits/76813b11d3b1383ef83bfea0e5c7781a986a2a8a)